pkgs/development/python-modules/invoice2data/default.nix

```diff
@@ -9,15 +9,17 @@ buildPythonPackage rec {
     sha256 = "0p3ka85n0qmdkrjffvqgcj3pbl002srrlcjdk8pyhhwg6b78d4l7";
   };
 
+  makeWrapperArgs = ["--prefix" "PATH" ":" "${tesseract}/bin/tesseract:${imagemagick}/bin/convert:${xpdf}/bin/pdftotext" ];
```

Comment:
This doesn't work, you need to just add the path to packages 'bin' directory, not the executable itself.
For multiple packages, this can also look like this:

```nix
makeWrapperArgs = ["--prefix" "PATH" ":" "${stdenv.lib.makeBinPath [ imagemagick xpdf ]}"];
```